news/2018-06-02-mappers-choice-awards-2017-results.md

```diff
@@ -0,0 +1,345 @@
+
+*written by [Maxus](https://osu.ppy.sh/u/4335785)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/sara.jpg)](https://osu.ppy.sh/s/683691)
+
+At first there were conflicting opinions towards this map because of its fierce SV uses and LN concepts, but finally it is accepted and ranked soon afterwards. Hardly ranked map have implementation of LN patterns throughout the whole map, while maps with fierce SVs are very rare as well. The existence of this map in ranked section remains very special because of its wonderful combination of both rarely-used elements, and 220 bpm also proposes a speed challenge to even those top players. Generally speaking, it's hard to believe that a map of anime song can be created in such an unorthodoxical way, thus winning it the Unorthodox category.
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/whiteout.jpg)](https://osu.ppy.sh/s/653740)
+
+SOUND VOLTEX has always been a loyal customer for both mania mappers and mania players, and this one too isn't an exception when we talking about rythm game category. **[Tofu1222](https://osu.ppy.sh/u/6089608)** manages to bring such an interesting and various LN layering concept into the map that suits with how the song incorporates lot of synth into it. Overall a really intriguing chart with a well deserved spot in this category.
+
+*written by [Maxus](https://osu.ppy.sh/u/4335785)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/swampgator.jpg)](https://osu.ppy.sh/s/613792)
+
+Inspired by **[Wonki](https://osu.ppy.sh/u/7898495)**'s first Zekk map, **[Trigger](https://osu.ppy.sh/s/555076)**, he created another popular mapset featuring quite aggressive SVs and jackhammers. In **[Swampgator](https://osu.ppy.sh/s/613792)**, we can see a development in the variation of SVs used, and how they fit with the patterns and the music in general to make it actually enjoyable to play. With an entry of Spotlight in that month, and quite a noticeable amount of favorites, it's well deserving of an award.
+
+*written by [Tofu1222](https://osu.ppy.sh/u/6089608)*
+
+[![](/wiki/shared/news/2018-06-02-mappers-choice-awards-2017-results/hitorigoto.jpg)](https://osu.ppy.sh/s/602861)
+
+Definitely one of the most popular song just by looking at the amount of plays that its standard counterpart has, and it's clearly not the exception for mania either. **[Tofu1222](https://osu.ppy.sh/u/6089608)** manages to bring such an unique touch with the mixing of the piano keysound that further develop the players enjoyment, and not stopping on that, he also uniquely bring his more unorthodox style of pattern approachs in the anime map by putting vocal pitch emphasis into kiai section by layering it into jackhammer. With multiple plus factor that he put into the map, he manages to bring himself into the top spot for the japanese category.
```

Comment:
popular songs just by looking at the number of plays
an unique -> a unique
of the piano keysound -> of piano keysounds
enjoyment, and not stopping on that, he -> enjoyment. In addition,
-> he also brought an unorthodox style of patterning for an anime map by putting emphasis on vocal pitch during the kiai section by layering it with jackhammers. 
With multiple plus factor -> Because of these positive factors (or something similar)
manages to bring himself into the top spot -> managed to earn himself the top spot
japanese -> Japanese